Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -9917,6 +9918,1640 @@
+													"                }\r",
+													"                else {\r",
+													"                    pm.expect(originalEntity[key], key).to.not.deep.equal(updatedEntity[key]);\r",
+													"                }\r",
+													"            }\r",
+													"        }  \r",
+													"    });\r",
+													"}"
+												],
+												"type": "text/javascript"
+											}
+										},
+										{
+											"listen": "prerequest",
+											"script": {
+												"exec": [
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"protocolProfileBehavior": {
+										"disabledSystemHeaders": {
+											"content-type": true
+										}
+									},
+									"request": {
+										"method": "PUT",
+										"header": [
+											{
+												"key": "Content-Type",
+												"value": "application/vnd.ed-fi.school.test-profile-resource-excludeonly.writable+json",
+												"type": "text"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\r\n  \"nameOfInstitution\": \"Modified\",\r\n  \"shortNameOfInstitution\": \"Modified\",\r\n  \"operationalStatusDescriptor\": \"uri://ed-fi.org/OperationalStatusDescriptor#Changed\",\r\n  \"website\": \"Modified\",\r\n  \"charterStatusDescriptor\": \"uri://ed-fi.org/CharterStatusDescriptor#School Charter\",\r\n  \"administrativeFundingControlDescriptor\": \"uri://ed-fi.org/AdministrativeFundingControlDescriptor#Private School\",\r\n  \"internationalAddresses\": [\r\n      {\r\n        \"addressTypeDescriptor\": \"uri://ed-fi.org/AddressTypeDescriptor#Other\",\r\n        \"countryDescriptor\": \"uri://ed-fi.org/CountryDescriptor#MY\",\r\n        \"addressLine1\": \"Modified\",\r\n        \"addressLine2\": \"Modified\",\r\n        \"addressLine3\": \"Modified\",\r\n        \"addressLine4\": \"Modified\"\r\n      }\r\n  ],\r\n  \"educationOrganizationCategories\": [\r\n    {\r\n      \"educationOrganizationCategoryDescriptor\": \"uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Other\"\r\n    }\r\n  ],\r\n  \"gradeLevels\": [\r\n    {\r\n      \"gradeLevelDescriptor\": \"uri://ed-fi.org/GradeLevelDescriptor#Kindergarten\"\r\n    }\r\n  ]\r\n}",
+											"options": {
+												"raw": {
+													"language": "json"
+												}
+											}
+										},
+										"url": {
+											"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/schools/{{known:intlSchoolId}}",
+											"host": [
+												"{{ApiBaseUrl}}"
+											],
+											"path": [
+												"data",
+												"v3",
+												"ed-fi",
+												"schools",
+												"{{known:intlSchoolId}}"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "The Write content type only excludes certain properties from derived association Copy",
```

Comment:
Check assertions " the persisted entity model should not have the new values assigned to the explicitly excluded properties" and "the only values not changed should be the explicitly excluded values, the id, and the primary key values" and also exclude 'Copy' from the name